### PR TITLE
[Do Not Merge] Update deprecate playground plugin

### PR DESCRIPTION
**Pull request by trevor-scheer**
This PR only exists as a simple mechanism to releasing a patch to our deprecated GraphQL Playground plugin. This should not land to `main`, as we don't want to actually restore the source code for this plugin.

### CODE REVIEW

File: .changeset/config.json [hunk 1]
```diff
@@ -5,7 +5,6 @@
     { "repo": "apollographql/apollo-server" }
   ],
   "commit": false,
-  "fixed": [["@apollo/server", "@apollo/server-integration-testsuite"]],
```
Comment (glasser) [inline .changeset/config.json:8]: Not sure what this does

File: packages/integration-testsuite/src/apolloServerTests.ts [hunk 2]
```diff
@@ -2777,24 +2776,6 @@ export function defineIntegrationTestSuiteApolloServerTests(
         );
       });
 
-      // We can remove this test after the initial AS4 release once we stop supporting
-      // the playground package.
-      it('can install playground with specific version', async () => {
```
Comment (glasser) [inline packages/integration-testsuite/src/apolloServerTests.ts:2782]: We should probably do this on main afterwards?

File: packages/plugin-landing-page-graphql-playground/README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # GraphQL Playground plugin
 
-> ⚠️ Note: this package will be completely unsupported when Apollo Server v4 is officially released. This package exists for migration purposes only. We do not intend to resolve security issues or other bugs with this package if they arise, so please migrate away from this to Apollo Server's default Explorer as soon as possible. ⚠️
+> ⚠️ Note: this package is unsupported. This package exists for v4 migration purposes only. We do not intend to resolve security issues or other bugs with this package if they arise, so please migrate away from this to Apollo Server's default Explorer as soon as possible. ⚠️
```
Comment (glasser) [inline packages/plugin-landing-page-graphql-playground/README.md:3]: I wonder if we should (a) link to https://www.apollographql.com/docs/apollo-server/previous-versions and (b) add something specific about this package there?

File: .changeset/config.json (hunk 1 above)
Comment (glasser) [reply .changeset/config.json:8]: (but i mean whatever happened on the other PR is fine so nbd)

File: packages/plugin-landing-page-graphql-playground/README.md (hunk 3 above)
Comment (trevor-scheer) [reply packages/plugin-landing-page-graphql-playground/README.md]: Updated for (a) but this package seems a bit specific to be called out in (b). If you have a specific recommendation for (b) I'll consider it?

File: .changeset/config.json (hunk 1 above)
Comment (trevor-scheer) [reply .changeset/config.json:8]: May or may not have been necessary, I flailed a bit trying to get the publish PR to only want to patch the plugin (and not the server + testsuite packages as well).

**Review by glasser — APPROVED** [review]
re the previous-versions page, there is a part that mentions graphql playground — could also just briefly note that the AS4 compatibility plugin is no longer supported either. Though it's in the AS2 section so that's a bit weird.

File: packages/integration-testsuite/src/apolloServerTests.ts (hunk 2 above)
Comment (trevor-scheer) [reply packages/integration-testsuite/src/apolloServerTests.ts:2782]: https://github.com/apollographql/apollo-server/pull/7586

**Comment by trevor-scheer** [comment]
The intent of this PR is complete, the newly published patch with README update [can be seen here](https://www.npmjs.com/package/@apollo/server-plugin-landing-page-graphql-playground).

**Pull request closed without merging**